recipes/dimspy/meta.yaml

```diff
@@ -1,42 +1,43 @@
 {% set name = "dimspy" %}
-{% set version = "1.4.0" %}
+{% set version = "2.0.0" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 890900f653b3131fc12e71bc4ca587513b0e19d7b1fad3783ffce6b9e590dac1
+  sha256: fbb908e6de6eac14e14caf56773970c44ec9970b8486074c602ee57c8dd86577
 
 build:
   noarch: python
   number: 0
   entry_points:
     - dimspy = dimspy.__main__:main
-
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+  
 requirements:
   host:
-    - python <3
+    - python >3.7
     - pip
-    - fastcluster ==1.1.23
-    - h5py ==2.7.0
-    - mono ==4.8.1.0
-    - numpy ==1.13.3
-    - plotly ==2.5.1
-    - pymzml ==0.7.10
-    - pythonnet ==2.4.0
-    - scipy ==1.1.0
+    - fastcluster==1.1.26
+    - h5py==2.10.0
+    - numpy==1.17.1
+    - pandas==0.25.0
+    - pymzml==2.4.5
+    - pythonnet==2.4.0
+    - pytables==3.6.1
+    - scipy==1.3.1
   run:
-    - python <3
-    - fastcluster ==1.1.23
-    - h5py ==2.7.0
-    - mono ==4.8.1.0
-    - numpy ==1.13.3
-    - plotly ==2.5.1
-    - pymzml ==0.7.10
-    - pythonnet ==2.4.0
-    - scipy ==1.1.0
+    - python >3.7
+    - fastcluster==1.1.26
```

Comment:
@RJMW is there a reason for such strict pins? I would remove all pins to be honest.